packages/shared/src/hooks/utils/useDownloadUrl.ts

```diff
@@ -0,0 +1,22 @@
+import { useMutation, MutationOptions } from '@tanstack/react-query';
```

Comment:
@capJavert do you think we still need this hook? Or we can just define the reusable function and then do `useMutation` on-demand?

Reply:
Since does not need anything from context/react scope that would require a hook, yes, having it as function sounds good as well 👍

Reply:
Made the changes here https://github.com/dailydotdev/apps/pull/2613/commits/3c779a73e5c04783c7b6e0c4d6f6f454b1a5ba50